src/plugins/oSnap/utils/tenderly.ts

```diff
@@ -53,3 +56,7 @@ export function prepareTenderlySimulationPayload(props: {
 
   return { safe: payload };
 }
+
+export function exceedsOsnapGasSubsidy(res: TenderlySimulationResult): boolean {
+  return res.gasUsed > OSNAP_GAS_SUBSIDY;
```

Comment:
Note that simulation gas usage might be lower than real one due to how state overrides short-circuit the OOv3 settlement logic. The execution bot uses ethers estimateGas on oSnap executeProposal call at the time when challenge window passed, so it might be worth checking this divergence in some test runs

Reply:
interesting we might want to implement some sort or buffer percentage.